.gofmt.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
sure. we'll just need to make sure we add new entries for any other golang subprojects we add to this repo, since go vet needs them to be specified to work.